conformance/tests/backendtlspolicy-observed-generation-bump.yaml

```diff
@@ -37,9 +37,5 @@ spec:
       name: "observed-generation-bump-test"
       sectionName: "https"
   validation:
-    caCertificateRefs:
-      - group: ""
-        kind: ConfigMap
-        # This ConfigMap is generated dynamically by the test suite.
-        name: "backend-tls-checks-certificate"
+    wellKnownCACertificates: System
```

Comment:
The test has nothing to do with whether caCertificateRefs or wellKnownCACertificates is used; just using wellKnownCACertificates to make a simple policy that should be Accepted. The purpose of the test does not depend on this.

We could also create the CM but its more complex for no benefit